### DISCUSSION TITLE
Privacy Manifest - Required Reason APIs

### DISCUSSION THREAD

**Discussion by nwelchbauer-disney** [adobe/aepsdk-core-ios#961] [General] [answered]
Starting Spring 2024, third-party iOS SDKs are required to include PrivacyInfo.xcprivacy manifests detailing reasons for usage of certain privacy-sensitive APIs or embedding apps will face App Review rejection.

https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

It looks like a number of the AEP iOS SDKs use UserDefaults, so the reason will need to be detailed in a PrivacyInfo.xcprivacy.

**Comment by isthisjoe** [comment]
+1

**Reply by svoip** [reply]
The deadline for the API declaration is getting nearer. Any news on this front?

**Comment by nwelchbauer-disney** [comment, marked as answer]
Version 5.0.0 includes a privacy manifest. Thanks!
